python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -87,6 +87,10 @@ class ImageDifferenceConfig(pexConfig.Config):
         dtype=bool,
         default=True
     )
+    astrometer = pexConfig.ConfigurableField(
+        target = measAstrom.AstrometryTask,
+        doc = "astrometry task; used to match sources to reference objects, but not to fit a WCS",
+    )
```

Comment:
It's for using hscAstrom instead of meas_astrom.  That shouldn't be necessary any more.  Don't worry about it.  If we need it for HSC, I'll work around you.